src/rdf4cpp/BlankNode.cpp

```diff
@@ -132,6 +132,20 @@ CowString BlankNode::identifier() const noexcept {
     return CowString{CowString::borrowed, handle_.bnode_backend().identifier};
 }
 
+FetchOrSerializeResult BlankNode::fetch_or_serialize_identifier(std::string_view &out, writer::BufWriterParts writer) const noexcept {
+    if (handle_.is_inlined()) {
+        auto const inlined = detail_bnode_inlining::from_inlined(handle_.id());
+        if (!rdf4cpp::writer::write_str(inlined.view().identifier, writer)) [[unlikely]] {
```

Comment:
How can this happen? Can You add a test case for this path?

Reply:
Can only happen if the writer has a finite amount of space or is unable to make additional space, so this has nothing to do with this PR

Reply:
would still be important to test it with every `fetch_or_serialize` function because not handling this case in any of them would make for very rare but hard to track down bugs.